### PR TITLE
Unify LV check logic

### PR DESCRIPTION
**Pull request by toshipp**
We have many lv check functions in e2e, it is less readable and maintainable, let's unify them.

### CODE REVIEW

File: e2e/helper_test.go [hunk 1]
```diff
@@ -138,39 +139,90 @@ func commonAfterEach(cc CleanupContext) {
...
-		size:   size * 512, // lvdisplay denotes size as 512 byte block
+		size:     size,
+		poolName: items[1],
+		vgName:   items[2],
 	}, nil
 }
+
+func checkLVIsRegisteredInLVM(volName string) error {
+	var lv topolvmv1.LogicalVolume
+	err := getObjects(&lv, "logicalvolumes", volName)
```
Comment (llamerada-jp) [inline e2e/helper_test.go:188]: `volName` makes me confuse. `volName` of checkLVIsDeletedInLVM meaning LVM's volume name, but `volName` here meaning LogicalVolume CR's name.

```suggestion
func checkLVIsRegisteredInLVM(lvResourceName string) error {
	var lv topolvmv1.LogicalVolume
	err := getObjects(&lv, "logicalvolumes", lvResourceName)
```

File: e2e/helper_test.go [hunk 2]
```diff
@@ -138,39 +139,90 @@ func commonAfterEach(cc CleanupContext) {
...
+	if err != nil {
+		if _, ok := err.(ErrLVNotFound); ok {
+			return nil
+		}
+		return err
+	}
+	return fmt.Errorf("target LV exists %s", volName)
+}
+
+func getLVNameofPVC(pvcName, ns string) (lvName string, err error) {
```
Comment (daichimukai) [inline e2e/helper_test.go:207]:
```suggestion
func getLVNameOfPVC(pvcName, ns string) (lvName string, err error) {
```
but this is not mandatory.

**Review by daichimukai — APPROVED** [review]

File: e2e/helper_test.go (hunk 1 above)
Comment (toshipp) [reply e2e/helper_test.go:188]: indeed.

File: e2e/helper_test.go (hunk 2 above)
Comment (toshipp) [reply e2e/helper_test.go]: did it.
https://github.com/topolvm/topolvm/compare/347aa38bfa761a3e8f062cacf3ccace5d2c2409e..2f96870bff6172e936be6e5f742cb92581e3590e

File: e2e/helper_test.go (hunk 1 above)
Comment (toshipp) [reply e2e/helper_test.go:188]: We use inconsistent names everywhere, but it seems to use `lvName` for lvm volume name and `volName` or `volumeName` for k8s objects. I change checkLVIsDeletedInLVM parameter name to `lvName` now.

Comment (toshipp) [reply e2e/helper_test.go:188]: https://github.com/topolvm/topolvm/compare/2f96870bff6172e936be6e5f742cb92581e3590e..fc6b630dbd3db0082a69915d6e4fd15aa208e7f1

**Review by llamerada-jp — APPROVED** [review]

File: e2e/helper_test.go (hunk 2 above)
Comment (daichimukai) [reply e2e/helper_test.go]: Thank you.

**Review by daichimukai — APPROVED** [review]

**Pull request merged**